Dockerfile.try

```diff
@@ -0,0 +1,110 @@
+FROM postgres:15-bullseye
+
+SHELL ["/bin/bash", "-c"]
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        gnupg \
+        lsb-release \
+        wget && \
+    sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' && \
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null && \
+    apt-get update -y -qq --fix-missing && \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        clang \
+        clang-11 \
+        gcc \
+        git \
+        jq \
+        libssl-dev \
+        llvm-11 \
+        make \
+        postgresql-server-dev-15 \
+        pkg-config && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set up permissions so that the postgres user can install the plrust plugin
+RUN chmod a+rwx `$(which pg_config) --pkglibdir` `$(which pg_config) --sharedir`/extension
```

Comment:
can we restore the permissions after `plrust` is installed?

I know this is just for people to try out, but we might should be good citizens where we can.

Reply:
I realized that I misinterpreted what you had asked. I believe I have a solution for this coming up in a commit.